.travis.yml

```diff
@@ -1,15 +1,15 @@
 sudo: false
+
 language: node_js
+
 node_js:
-  - "0.10"
-  - "0.12"
-  - "4"
-  - "5"
-  - "iojs"
-before_install:
-  - if [ "$TRAVIS_NODE_VERSION" = "0.10" ]; then npm install -g npm@2; fi
+  - "6"
+  - "8"
+  - "10"
+
```

Comment:
Since 12 has been released, did we want to include that here?

Reply:
Yeah, when I made the branch, grunt-contrib-internal v4.0.0 wasn't published yet.